src/CashuMint.ts

```diff
@@ -48,7 +48,28 @@ class CashuMint {
 		customRequest?: typeof request
 	): Promise<GetInfoResponse> {
 		const requestInstance = customRequest || request;
-		return requestInstance<GetInfoResponse>({ endpoint: joinUrls(mintUrl, '/v1/info') });
+		const data = await requestInstance<GetInfoResponse>({
+			endpoint: joinUrls(mintUrl, '/v1/info')
+		});
+		// BEGIN DEPRECATED
+		// Monkey patch old contact field ["email", "me@mail.com"] Array<[string, string]>; to new contact field [{method: "email", info: "me@mail.com"}] Array<MintContactInfo>
+		// This is to maintain backwards compatibility with older versions of the mint
+		if (Array.isArray(data?.contact) && data?.contact.length > 0) {
+			data.contact = data.contact.map((contact: any) => {
```

Comment:
It would be better to use the `unknown` type instead of `any` for the `contact` parameter in the map function. The `unknown` type is safer because it requires us to explicitly check the type before performing operations on the variable, thus avoiding potential runtime errors. Here’s how you can update the code:

```typescript
if (Array.isArray(data?.contact) && data?.contact.length > 0) { 
    data.contact = data.contact.map((contact: unknown) => {
        // Perform necessary type checks or type assertions here
    });
}
```
---

Reply:
I think this should be fixed in [ff1f316](https://github.com/cashubtc/cashu-ts/pull/147/commits/ff1f3163ac99c32b2d0f0b799da257ab86d2251c)